core/test/qa_buffer.cpp

```diff
@@ -76,15 +76,15 @@ const boost::ut::suite BasicConceptsTests = [] {
                 expect(nothrow([&writer] { expect(writer.try_publish([](const std::span<int32_t> &, std::int64_t) { /* noop */ }, 0)); }));
 
                 // alt expert write interface
-                auto value = writer.reserve(1);
-                expect(eq(1LU, value.size())) << "for " << typeName;
-                if constexpr (requires { value.publish(1); }) {
-                    value.publish(1);
-                }
-                consumableInputRangeTest1(data);
-                consumableInputRangeTest2(data);
-                // consumableInputRangeTest3(data); // this does not compile
-                consumableInputRangeTest3(static_cast<std::span<const int32_t>>(data));
+                //{
+                //    auto value = writer.reserve(1);
+                //    expect(eq(1LU, value.size())) << "for " << typeName;
+                //    value.publish(1);
+                //}
+                //consumableInputRangeTest1(data);
+                //consumableInputRangeTest2(data);
+                //// consumableInputRangeTest3(data); // this does not compile
+                //consumableInputRangeTest3(static_cast<std::span<const int32_t>>(data));
```

Comment:
Is this code is still valid? If not please remove.

Reply:
reenabled, seems to only have caused trouble in some intermediate state. Thanks for catching this!